src/views/CancelTransfer.js

```diff
@@ -56,20 +57,18 @@ class CancelTransfer extends React.Component {
     const canGenerate = true;
 
     return (
-      <Row>
-        <Col>
-          <H1>
-            {'Cancel Transfer of '}{' '}
-            <code>{` ${ob.patp(state.pointInTransfer)} `}</code>
-          </H1>
-
-          <P>{`This action will cancel the transfer to ${proxy}.`}</P>
-          <StatelessTransaction
-            canGenerate={canGenerate}
-            createUnsignedTxn={this.createUnsignedTxn}
-          />
-        </Col>
-      </Row>
+      <View>
+        <H1>
+          {'Cancel Transfer of '}{' '}
```

Comment:
(this was already here but) what's up with this? No reason for this to exist, right? (The extra `{' '}`, that is.)

Reply:
in some cases you need a `{' '}` to emulate a `&nbsp;` (nonbreaking space character) because jsx will collapse new lines when rendering. There are a _ton_ of uses of `<div>{'Some text'}</div>` in the codebase where `<div>Some text</div>` is semantically identical, so I'm not sure how/when/why that pattern was adopted. I'll be changing them as I go and will make sure I don't break any expectations the UI has for these spaces.